server/gamenode/gameserver.js

```diff
@@ -353,7 +353,7 @@ class GameServer {
         }
 
         this.runAndCatchErrors(game, () => {
-            game.stopClocks();
+            game.stopNonChessClocks();
```

Comment:
This is what would cause the clocks to stop & restart every time a game state update happened (which would make it so you had an infinite buffer by clicking on blank cards).  Since chess clocks can work entirely off the UI prompts, I made this only apply to non-chess clocks.